fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewQualityCheckBottomSheet.kt

```diff
@@ -55,6 +55,9 @@ fun ReviewQualityCheckBottomSheet(
                         onRequestDismiss()
                         store.dispatch(ReviewQualityCheckAction.OptOut)
                     },
+                    onReanalyzeClick = {
+                        // TODO(bug 1847022)
```

Comment:
I haven't checked yet, but is the linter flagging the `todo`? I think we typically try to avoid those, but wasn't sure if the linter already had a rule in place 

(no need for actual changes if it's passing the linter. we'll be plugging in the real code soon)

Reply:
I went ahead and removed the todo, I think I misunderstood a previous comment of yours, but we really shouldn't be adding bug todos anyway.